File: 1672-richest-customer-wealth.py
"""
1676. Richest Customer Wealth

Submitted: October 30, 2024

Runtime: 0 ms (beats 100.00%)
Memory: 16.65 MB (beats 64.16%)
"""

class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:
        return max(sum(customer) for customer in accounts)

File: 1672-richest-customer-wealth.cpp
/*
1672. Richest Customer Wealth

Submitted: October 23, 2024

Runtime: 0 ms (beats 100.00%)
Memory: 10.26 MB (beats 98.97%)
*/

class Solution {
public:
    int maximumWealth(vector<vector<int>>& accounts) {
        int maxCount = 0;
        for (const vector<int>& account : accounts) {
            int sum = 0;
            for (const int& j : account) {
                sum += j;
            }
            if (sum > maxCount) {
                maxCount = sum;
            }
        }
        return maxCount;
    }
};
